### PR TITLE
Deprecate Picus Community

### PR DESCRIPTION
**Pull request by edik24**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-8005) , [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-8023)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by RotemAmit — COMMENTED** [review]
Great job!

File: Packs/PicusNGAutomation/ReleaseNotes/1_0_9.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Picus Security NG
+- Updated integration display name
```
[the remark this answers is not in the document]
Comment (edik24) [reply Packs/PicusNGAutomation/ReleaseNotes/1_0_9.md:3]: done

File: Packs/PicusNGAutomation/ReleaseNotes/1_0_9.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (RotemAmit) [inline Packs/PicusNGAutomation/ReleaseNotes/1_0_9.md:3]:
```suggestion
- Updated the integration display name.
```

File: Packs/PICUS/ReleaseNotes/1_0_4.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### PICUS (Deprecated)
+- Deprecated pack
```
Comment (RotemAmit) [inline Packs/PICUS/ReleaseNotes/1_0_4.md:3]:
```suggestion
- Deprecated the pack.
```

**Review by RotemAmit — COMMENTED** [review]
Please update the release notes, and I will approve the PR.
Nice work :)

**Review by RotemAmit — APPROVED** [review]

**Pull request merged**